### PR TITLE
Add rpm action

### PR DESCRIPTION
**Pull request by aconitumnapellus**
##### SUMMARY
This PR adds a github action that is used to build RPM packages. 
It will produce artifacts that can be downloaded, unzipped and installed.

##### ISSUE TYPE
 - Feature Pull Request

##### ADDITIONAL INFORMATION
This action does NOT automatically produce a release. Releases need to be produced manually, the artifact can then be attached to those.

### CODE REVIEW

File: nfpm.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+# Architecture. (required)
+arch: noarch
+
+# Platform.
+# Defaults to `linux`.
+platform: linux
+
+# Version. (required)
+# This will expand any env var you set in the field, eg version: v${SEMVER}
+version: v2.0.4
```
Comment (s3lph) [inline nfpm.yaml:15]: Is there any better what than hardcoding the version in the nfpm config? I can tell from personal experience that something like updating this version number gets forgotten a lot. In fact, this version number doesn't match then one in `backupctl/version.py`

Is the version available in some env var? E.g. before running nfpm, you could e.g. run `export BACKUPCTL_VERSION=$(python -c 'import backupctl.version; print(backupctl.version.__version__)')` and pass `-e BACKUPCTL_VERSION` to the docker nfpm run, and put `version: v${BACKUPCTL_VERSION}` here. (UNTESTED)

**Review by s3lph — CHANGES_REQUESTED** [review]
Looks mostly good, and pipeline seems to run successfully. However, please have a look at the issue regarding version numbers.

File: nfpm.yaml (hunk 1 above)
Comment (aconitumnapellus) [reply nfpm.yaml]: Ive set the version of the package once in the action.yml, i dont like it to be the source of the version, i might need to move it elsewhere, if you have an idea where, im open for ideas.

Comment (aconitumnapellus) [reply nfpm.yaml]: I added a version file with just the version in it and removed the "env" keyword.
Seems more clear to me.

**Review by s3lph — APPROVED** [review]

**Pull request merged**